src/materials_processor/houdini/output_detector.py

```
@@ -0,0 +1,284 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+try:
+    import hou
+except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)